### DISCUSSION TITLE
Translation Manager System

### DISCUSSION THREAD

**Discussion by chenlung** [URUWorks/TeroSubtitler#65] [Ideas]
Similar to [MuseScore](https://github.com/musescore/MuseScore), I think we should have a [translation manager system](https://en.wikipedia.org/wiki/Translation_management_system).

[Transifex](https://www.transifex.com) is one solution, but it's proprietary and costs money, so [here](https://alternativeto.net/software/transifex/?license=opensource) is a list of open-source ones.

**Comment by chenlung** [comment]
I'm thinking it would be a good idea to implement this before the beta stage commences in a few weeks.

Betas are probably the best time to ask translators to contribute because of fewer changes in relation to strings.

**Comment by chenlung** [comment]
One advantage might be not having to wait until there is a new version (strings can be updated within the release). I think this happens in MuseScore.

**Comment by GitGianluc** [comment]
> [Transifex](https://www.transifex.com) is one solution, but it's proprietary and costs money, so [here](https://alternativeto.net/software/transifex/?license=opensource) is a list of open-source ones.

Are we certain that Transifex is a paid service?

<img width="1089" alt="Screenshot 2023-06-06 alle 12 13 45" src="https://github.com/URUWorks/TeroSubtitler/assets/16348719/e8716eed-5ce1-4e64-9bc3-468384430830">

https://help.transifex.com/en/articles/6236788-open-source-projects

**Reply by chenlung** [reply]
I didn't know that about open source. Thanks for pointing that out!
